### PR TITLE
[FEATURE] intital version

### PR DESCRIPTION
**Pull request by aatauil**
## ID
  DL-5280

 ## Description

This PR adds all elements needed for filtering in the frontend. We used the search-query-management service and ember-submission-form-fields to generate the filtering form. 

 ## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Other

 ## Related services

 - [search-query-management-service](https://github.com/lblod/toezicht-search-query-management-service)

 ## How to test

This PR should be tested together with the following frontend PR:
- https://github.com/lblod/frontend-subsidiedatabank/pull/2

To test:
- setup app and frontend on correct branches
- wait for migrations to run 
- wait for delta-consumer to consume all data from loket
- mock login to frontend
- test all filters and see if they work

 ## Links to other PR's

 - https://github.com/lblod/frontend-subsidiedatabank/pull/2

### CODE REVIEW

File: config/search-query/filter-form.ttl [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    search:emberQueryParameterKey "bestuurseenheden" ;
+    form:options  """{"conceptScheme":"http://lblod.data.gift/concept-schemes/7e2b965e-c824-474f-b5d5-b1c115740083"}""" ;
+    form:help "vb. Gemeente Hasselt, OCMW Gent" ;
+    sh:order 20;
+    sh:group ext:applicationPg .
+
+ext:aanvraagDatumF a form:Field ;
+    sh:name "Datum aanvraag" ;
+    sh:path searchSubsidie:aanvraagDatum;
+    form:displayType displayTypes:date;
```
Comment (wolfderechter) [inline config/search-query/filter-form.ttl]: Should we use a datepicker here to make it easier for the user?
```suggestion
    form:displayType displayTypes:datePicker;
```

File: config/search-query/filter-form.ttl [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+##########################################################
+ext:typeSubsidieF a form:Field ;
+    sh:name "Subsidie" ;
+    sh:path searchSubsidie:subsidieMeasureOffer;
+    form:displayType displayTypes:conceptSchemeSelector ;
+    search:emberQueryParameterKey "subsidietype" ;
+    form:options  """{"conceptScheme":"http://lblod.data.gift/concept-schemes/5979ff3e-1f4c-4271-ae40-e1d2000d6412"}""" ;
+    form:help "vb. Contact-en bronopsporing, Fietsinfrastructuur" ;
+    sh:order 10;
+    sh:group ext:applicationPg .
```
Comment (wolfderechter) [inline config/search-query/filter-form.ttl]: The figma mockup has placeholders for the filters, which are currently not present in the frontend. Not sure if it's necessary

Comment (wolfderechter) [reply config/search-query/filter-form.ttl]: https://github.com/lblod/ember-submission-form-fields/blob/master/addon/components/rdf-input-fields/concept-scheme-selector.hbs if we add a placeholder attribute here, it should be pretty easy

File: config/search-query/filter-form.ttl (hunk 1 above)
Comment (aatauil) [reply config/search-query/filter-form.ttl]: I like this improvement :)

File: config/search-query/filter-form.ttl (hunk 2 above)
Comment (aatauil) [reply config/search-query/filter-form.ttl]: I think they are not really needed in this case. We have both a label & helptext so placeholder might be overkill. Also I dont think it is supported by ember-submission-form-fields currently

File: config/search-query/filter-form.ttl [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+ext:typeBestuurF a form:Field ;
+    sh:name "Type bestuur" ;
+    sh:path searchSubsidie:typeBestuur;
+    form:displayType displayTypes:conceptSchemeSelector ;
+    search:emberQueryParameterKey "bestuursType" ;
+    form:options  """{"conceptScheme":"http://data.vlaanderen.be/id/conceptscheme/BestuurseenheidClassificatieCode"}""" ;
+    form:help "vb. Gemeente Hasselt, OCMW Gent" ;
```
Comment (wolfderechter) [inline config/search-query/filter-form.ttl]: helptext was a duplicate of bestuurseenheid and should be changed
```suggestion
    form:help "vb. Gemeente, OCMW" ;
```

**Pull request merged**